File: Data Structures & Algorithms/duplicate-integer/submission-2.cs
public class Solution {
    public bool hasDuplicate(int[] nums) {
        bool flag=false;
        for(int i=0;i<nums.Length;i++){
            for(int j=0;j<nums.Length;j++){
                if(i!=j){
                    if(nums[i]==nums[j]){
                        flag=true;
                        return flag;
                    }
                    else{
                        flag=false;
                    }
                }
            }
        }
        return flag;
    }
}

File: Data Structures & Algorithms/majority-element/submission-2.py
class Solution:
    def majorityElement(self, nums: List[int]) -> int:
        mapNum={}
       
        for i in (set(nums)):
            print(i)
            maximum=nums.count(i)
            print(maximum)
            #val=next(maximum)
            #mapNum[i]=(maximum//2)
            mapNum[i]=maximum
        print(mapNum)
        return max(mapNum, key=mapNum.get)
